widgetssdk/src/main/java/com/glia/widgets/survey/SurveyAdapter.java

```diff
@@ -433,7 +422,7 @@ void showRequiredError(boolean error) {
                 ColorStateList strokeColor =
                         error ? ContextCompat.getColorStateList(context, R.color.glia_system_negative_color) :
                                 ContextCompat.getColorStateList(context, R.color.glia_base_shade_color);
-                int width = (int) Math.round(context.getResources().getDimension(R.dimen.glia_px));
+                int width = Math.round(context.getResources().getDimension(R.dimen.glia_px));
```

Comment:
better way to fetch dimensions is using `context.getResources().getDimensionPixelSize(R.dimen.glia_px)`